package.json

```diff
@@ -15,7 +15,8 @@
     "test:diff": "mocha './test/**/*.visual-diff.js' -t 10000",
     "test:diff:golden": "mocha './test/**/*.visual-diff.js' -t 10000 --golden",
     "test:local": "polymer test --skip-plugin sauce",
-    "test:sauce": "polymer test --skip-plugin local"
+    "test:sauce": "polymer test --skip-plugin local",
+    "test:travis": "polymer test --config-file wct.conf-travis.json"
```

Comment:
I think `--skip-plugin` can take a list of plugins... so in theory if you didn't want to introduce the config file you could create a 3rd plugin in the `wct.conf.json` file just for travis.

Reply:
I think the plugins are more "official" than that - I get an error `Could not find WCT plugin named "travis"`. Looking into if there's a way around that.

Reply:
Yeah there doesn't seem to be a way to get custom plugin names that I have found

Reply:
OK, thanks for trying!